### PR TITLE
db/eslint-dashboards

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added a separate eslint config for dashboards.

### CODE REVIEW

**Review by goransle — COMMENTED** [review]
If we're using tsdoc, should we add https://www.npmjs.com/package/eslint-plugin-tsdoc as well here?

File: ts/Dashboards/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../.eslintrc",
+    "rules": {
+      "jsdoc/require-description": 0,
+      "jsdoc/require-jsdoc": 0,
```
Comment (bre1470) [inline ts/Dashboards/.eslintrc:5]: These still have to be warnings as the source code has to be documented.

**Comment by bre1470** [comment]
> If we're using tsdoc, should we add https://www.npmjs.com/package/eslint-plugin-tsdoc as well here?

Does TypeDoc use TSDoc?

**Comment by goransle** [comment]
> > If we're using tsdoc, should we add https://www.npmjs.com/package/eslint-plugin-tsdoc as well here?
> 
> Does TypeDoc use TSDoc?

Looks like yes https://typedoc.org/guides/doccomments/#tsdoc-compliance. But I was actually confused by the similar names 😓

**Comment by bre1470** [comment]
> > > If we're using tsdoc, should we add https://www.npmjs.com/package/eslint-plugin-tsdoc as well here?
> > 
> > 
> > Does TypeDoc use TSDoc?
> 
> Looks like kind of sort of yes https://typedoc.org/guides/doccomments/#tsdoc-compliance. But I was actually confused by the similar names 😓

Hmm, they write: "TypeDoc aims to be compliant with the TSDoc standard, but does not enforce it. This means that while TypeDoc should be able to parse all (or nearly all) TSDoc-complaint comments, it does not require that your comments follow the standard."

For flexibility we should keep the setup simple. It is a temporary solution anyway.

**Review by pawellysy — COMMENTED** [review]
Not relevant maybe we could: 
```json
"max-len": {
    "code": 120
    }
 ```
 ? 😅

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by pawellysy — APPROVED** [review]
LGTM!

**Pull request merged**